app/components/error_summary_component/view.html.erb

```diff
@@ -0,0 +1,18 @@
+<% if @errors.any? %>
```

Comment:
We shouldn't need this check if we add a method called `render?` and add the condition there

```
def render?
  @errors.any?
end
```